lib/src/native/realm_core.dart

```diff
@@ -1097,7 +1104,7 @@ class _RealmCore {
     return _realmLib.realm_app_get_app_id(app.handle._pointer).cast<Utf8>().toRealmDartString()!;
   }
 
-  static void _app_user_completion_callback(Pointer<Void> userdata, Pointer<realm_user> user, Pointer<realm_app_error> error) {
+  static void app_user_completion_callback(Pointer<Void> userdata, Pointer<realm_user> user, Pointer<realm_app_error> error) {
```

Comment:
We should be careful when making statements like this. This is not polluting the PR. This change brings consistency across the callback names. Which I noticed while working on the changes. I don't think it matters much if we use underscore or not and since we had callbacks that don't use it I decided to rename these as well. There should be no real problem here.